### DISCUSSION TITLE
monitoring/azurecli/how-to-remove-azure-log-monitoring-agent-mma-or-oms-with-azure-cli/

### DISCUSSION THREAD

**Discussion by giscus** [eloy-salamanca/eloy-salamanca.github.io#31] [Announcements]
# monitoring/azurecli/how-to-remove-azure-log-monitoring-agent-mma-or-oms-with-azure-cli/

Options to remove Azure Log Monitoring Agent (MMA or OMS) individually and in bulk using Azure CLI

https://eloysalamanca.es/monitoring/azurecli/how-to-remove-azure-log-monitoring-agent-mma-or-oms-with-azure-cli/

**Comment by pixpoxstudy** [comment]
how can I read using ephemeral AZ cli session to pass all resources groups?

**Reply by eloy-salamanca** [reply]
Hi @pixpoxstudy,

Yeah, that’s the next step, to chain different resource groups all over, but honestly, I haven't tried that one yet! Do you dare?
